packages/agents/sequencer/src/lib/helpers/relayerfee.ts

```diff
@@ -49,8 +53,8 @@ export const canSubmitToRelayer = async (transfer: XTransfer): Promise<{ canSubm
   let relayerFeePaidUsd = constants.Zero;
   for (const asset of relayerFeeAssets) {
     if (asset === constants.AddressZero) {
-      const destChainId = domainToChainId(+destinationDomain);
-      const nativeUsd = await getConversionRate(destChainId, undefined, logger);
+      const originChainId = await domainToChainId(+originDomain);
```

Comment:
No await here anymore. This is changed to using the sync versions.
```suggestion
      const originChainId = domainToChainId(+originDomain);
```

Reply:
right

Reply:
https://github.com/connext/monorepo/pull/3798/commits/063836cf00769a093e4daa9abf4529229b7ebc70
fixed here